exonum/src/blockchain/schema.rs

```diff
@@ -273,15 +274,15 @@ where
         sum_table.get_proof(&key)
     }
 
-    fn find_configurations_index_by_height(&self, height: u64) -> u64 {
+    fn find_configurations_index_by_height(&self, height: Height) -> u64 {
         let actual_from = self.configs_actual_from();
         for i in (0..actual_from.len()).rev() {
-            if actual_from.get(i).unwrap().actual_from() <= height {
+            if actual_from.get(i).unwrap().actual_from() <= height.0 {
                 return i as u64;
             }
         }
         panic!(
-            "Couldn't not find any config for height {}, \
+            "Couldn't not find any config for height {:?}, \
```

Comment:
Actually, `Display` is already implemented for `Height`, I just forgot to update usage.